edgelet/contrib/enterprise-linux/aziot-edge.spec

```diff
@@ -0,0 +1,180 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/run/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+%define iotedge_agent_user edgeagentuser
+%define iotedge_agent_uid 13622
+%global debug_package %{nil}
+
+Name:           aziot-edge
+Version:        @version@
+Release:        @release@%{?dist}
+
+License:        MIT
+Summary:        Azure IoT Edge Module Runtime
+URL:            https://github.com/azure/iotedge
+
+%{?systemd_requires}
+BuildRequires:  systemd
+Requires(pre):  shadow-utils
+Requires:       aziot-identity-service = 1.3.0~dev-1
```

Comment:
Please create a PBI to update the release procedure to update this file for upcoming releases.